pyintegration/deephaven2/stream/kafka/consumer.py

```diff
@@ -55,9 +56,9 @@ class TableType(Enum):
 
 
 class KeyValueSpec(JObjectWrapper):
-    _j_spec: jpy.JType
```

Comment:
If `_j_spec` still exists, should this type annotation still exist?

Reply:
strictly speaking that's a class member, just having the same name for both a class variable and a member variable is somewhat bug prone at times.